src/components/Segmented/Segmented.tsx

```diff
@@ -0,0 +1,143 @@
+import { ReactElement, useState } from 'react'
+import classnames from 'classnames'
+
+import { Option, OptionsAlignments } from './Segmented.types'
+import { getOptionKey, isDisabledOption, isString, isVerticalOption, resolveKey } from './Segmented.utils'
+import styles from './Segmented.module.css'
+
+const { Horizontal, Vertical } = OptionsAlignments
+const { segmented, segmentedOption, disabled, selected, vertical } = styles
+
+export type SegmentedProps = {
+
```

Comment:
Didn't we decide to remove these new lines?

Reply:
We already removed the padded blocks rule. This space is different, it is enforced because it is a comment (eslint rule lines-around-comment) which comes with the recommended default configuration. If we want to remove this space we need to change that rule. Why does this space represent a problem to you?

Reply:
I agree with Fede, let's remove these lines. They don't help that much with readability and I would start thinking about putting the types in a separate file, what do you think?

Reply:
Regarding the name of the component instead, it is a design choice. Nothing to do with the work expressed in this merge request. It is also a very typical name for this kind of interaction.

Reply:
I just found out that antd has the same kind of component with the same name but it's the first time I see a button group with that name tbh, maybe "SegmentedButton" would be a clearer name?

Reply:
To better address the name issue I've done a little bit of research:

* antd calls this component "segmented" https://ant.design/components/segmented 
* many other calls it segment controls:
   * uber: https://base.uber.com/6d2425e9f/p/05421e-segmented-control/b/46994b
   * apple: https://developer.apple.com/design/human-interface-guidelines/segmented-controls
   * gitlab: https://design.gitlab.com/components/segmented-control
   * microsoft: https://fluent2.microsoft.design/components/ios/segmentedcontrol/usage

Reply:
The most used name is https://component.gallery/components/segmented-control/ SegmentedControls. Can it be called like this?

Reply:
Discussing with the design team, the final decision for the component name is *Segmented Control*